### PR TITLE
Asio, Libuv and Epoll are now inside uS namespace

### PR DESCRIPTION
**Pull request by ymikhailov**

### CODE REVIEW

File: src/Hub.h [hunk 1]
```diff
@@ -41,7 +41,7 @@ struct WIN32_EXPORT Hub : private uS::Node, public Group<SERVER>, public Group<C
     bool listen(int port, uS::TLS::Context sslContext = nullptr, int options = 0, Group<SERVER> *eh = nullptr);
     bool listen(const char *host, int port, uS::TLS::Context sslContext = nullptr, int options = 0, Group<SERVER> *eh = nullptr);
     void connect(std::string uri, void *user = nullptr, std::map<std::string, std::string> extraHeaders = {}, int timeoutMs = 5000, Group<CLIENT> *eh = nullptr);
-    void upgrade(uv_os_sock_t fd, const char *secKey, SSL *ssl, const char *extensions, size_t extensionsLength, const char *subprotocol, size_t subprotocolLength, Group<SERVER> *serverGroup = nullptr);
+    void upgrade(uS::uv_os_sock_t fd, const char *secKey, SSL *ssl, const char *extensions, size_t extensionsLength, const char *subprotocol, size_t subprotocolLength, Group<SERVER> *serverGroup = nullptr);
```
Comment (alexhultman) [inline src/Hub.h]: This probably has to be uv_os_sock_t

File: src/Hub.cpp [hunk 2]
```diff
@@ -198,14 +198,14 @@ void Hub::upgrade(uv_os_sock_t fd, const char *secKey, SSL *ssl, const char *ext
     // todo: skip httpSocket -> it cannot fail anyways!
     HttpSocket<SERVER> *httpSocket = new HttpSocket<SERVER>(&s);
     httpSocket->setState<HttpSocket<SERVER>>();
-    httpSocket->change(httpSocket->nodeData->loop, httpSocket, httpSocket->setPoll(UV_READABLE));
+    httpSocket->change(httpSocket->nodeData->loop, httpSocket, httpSocket->setPoll(uS::UV_READABLE));
```
Comment (alexhultman) [inline src/Hub.cpp]: This also probably has to be UV_READABLE

**Comment by alexhultman** [comment]
Basically everything that is normally not namespaced in libuv should not be namespaced here either (UV_READABLE, uv_os_sock_t, etc). Wrappers like Poll are correctly uS-namespaced.

**Comment by ymikhailov** [comment]
Done. Now uv_sock_t, UV_READABLE and UV_WRITABLE in global namespace again.

**Pull request merged**